### DISCUSSION TITLE
Select/Multiselect default value

### DISCUSSION THREAD

**Discussion by mohammed-mahmoud-scandi** [pimcore/pimcore#15617] [Q&A / Support]
Is there's any intention to set default values for multiselect data type like for select ?

**Comment by kingjia90** [comment]
Thank you for your message! 
I've checked and it seems it was never implemented, please follow the issue https://github.com/pimcore/pimcore/issues/15642 to see any progress in there on when it would be done (release/milestone etc..).

Closing here for the moment
